Automattic/wp-calypso
# Stats: Adding UTM upgrade overlay

Related to #87571

## Proposed Changes

* adding and overlay to the UTM module for plans that don't support it
* proper visibility will be adjusted as part of https://github.com/Automattic/wp-calypso/issues/87569

## Testing Instructions



* open the live branch
* create new JN site and instead of purchasing a plan click "I will do it later"
* go to the Stats on Calypso for that JN and apply `stats/utm-module` featureflag
* verify that the new UTM module is covered with an overlay (screenshot below)
* extra: purchase a commercial plan for a new JN page and verify that you see the component without an overlay

![SCR-20240223-oqyz](https://github.com/Automattic/wp-calypso/assets/112354940/1b496a0e-60ac-4f83-a29e-1aec91b2f481)

## Pre-merge Checklist
